src/glue_launcher_lambda/glue_launcher.py

```diff
@@ -355,11 +363,15 @@ def clear_manifest_output(bucket, prefix):
 
         # flush once aws limit reached
         if len(key_to_delete["Objects"]) >= 1000:
+            logger.info(f"Deleting 1000 objects")
```

Comment:
Establish the count of items you are deleting - otherwise this log will be confusing and misleading.

Reply:
```suggestion
            logger.info(f"Deleting '{len(key_to_delete["Objects"])}' objects")
```